### DISCUSSION TITLE
is there any delay to showing up Wikidata when editing OSM via iD once an item was already added into NSI and displaying on https://nsi.guide

### DISCUSSION THREAD

**Discussion by ydwangd** [osmlab/name-suggestion-index#6592] [General]
"杭州公共自行车" or "Hangzhou Public Bicycle" can be displayed on https://nsi.guide/ when I click OSM name suggestion index id in its Wikidata but when I trying to use it in OSM editing no lucky not displaying in OSM feature  search bar

**Comment by LaoshuBaby** [comment]
Hey, data in https://nsi.guide will be updated when someone conduct `npm run dist`, and data in iD or other editor related to npm package. Only after https://www.npmjs.com/package/name-suggestion-index release a new version you can use it in production environment editor. Release checklist contain a `npm run dist`.

https://github.com/osmlab/name-suggestion-index/commit/8a7e98cb169d639a77b7b14c899da259c36f4258 （6.0.20220516）

**↑** Latest version is released 7 hours ago, but id editor is using jsdeliver's cdn, and the cdn haven't refresh to latest version:

![20220509](https://user-images.githubusercontent.com/42690037/168550649-bb96e138-93db-48ce-8bf2-c70c5f8192d9.png)

**Reply by ydwangd** [reply]
ok thank you
